API/controllers/user-controller.js

```diff
@@ -0,0 +1,8 @@
+const { fetchUsers } = require("../models/user-model");
+
+exports.getUsers = (req, res, next) => {
+  return fetchUsers().then(({ rows }) => {
+    const users = rows;
```

Comment:
This could have been handled in the model, but not overly a problem to leave it as is.